app/controllers/sessions_controller.rb

```diff
@@ -14,8 +14,27 @@ def create
     end
   end
 
+  def update
+    auth_hash = request.env['omniauth.auth']
+    current_user.update(github_token: auth_hash[:credentials][:token])
+    redirect_to dashboard_path
+  end
+
   def destroy
     session[:user_id] = nil
     redirect_to root_path
   end
+
+  private
```

Comment:
This was originally in a private method after reading about it on a blog about Omniauth. We can refactor later and move it back into a private method.

Reply:
Sounds good, looks like it should work either way.